src/CONST.js

```diff
@@ -2,6 +2,7 @@ const CLOUDFRONT_URL = 'https://d2k5nsl2zxldvw.cloudfront.net';
 
 const CONST = {
     // 50 megabytes in bytes
+    ANIMATED_TRANSITION: 300,
```

Comment:
Can you switch the order of these lines please? `// 50 megabytes in bytes` is important to the following const.